gxcloudstorage-awss3-v2/src/main/java/com/genexus/db/driver/ExternalProviderS3V2.java

```diff
@@ -95,6 +98,9 @@ private void initialize() throws Exception {
 		String bucket = getEncryptedPropertyValue(BUCKET, BUCKET_DEPRECATED);
 		String folder = getPropertyValue(FOLDER, FOLDER_DEPRECATED, "");
 		clientRegion = getPropertyValue(REGION, REGION_DEPRECATED, DEFAULT_REGION);
+
+		objectOwnershipEnabled = !getPropertyValue(DEFAULT_ACL, DEFAULT_STORAGE_PRIVACY, "").contains("Bucket owner enforced");
```

Comment:
Can be refactor this String? 
"Bucket owner enforced"

Is used twice in the code. Maybe we can change it as a Constant.

Reply:
Implemented